docs/getting-started.md

```diff
@@ -0,0 +1,119 @@
+# Getting Started
+
+Before diving into Langium itself, let's get your environment ready for development:
+
+1. You have a working [Node environment](https://nodejs.org/en/download/) with version 12 or higher.
+2. Install Yeoman and the Langium extension generator.
+```bash
+npm i -g yo generator-langium
+```
+
+For our getting started example, we would also recommend you to install the latest version of [vscode](https://code.visualstudio.com/).
+
+## Your first example language
+
+To create your first working DSL, execute the yeoman generator:
+
+```bash
+yo langium
+```
+
+Yeoman will prompt you with a few basic questions about your DSL:
+
+1. _Extension name_: Will be used as the folder name of your extension and its `package.json`.
+2. _Language name_: Will be used as the name of the grammar and as a prefix for any generated files. 
+3. _File extensions_: A comma separated list of file extensions for your DSL.
+
+Afterwards, it will generate a new project and start installing all dependencies, including the `langium` framework as well as the `langium-cli` command line tool required for generating code based on your grammar definition.
+
+After everything has successfully finished running, open your newly created Langium project with vscode (replacing `hello-world` with your chosen project name):
```

Comment:
Explain that you can either open the project via the UI (File > Open...) or the following command.